subsys/bluetooth/controller/ll_sw/ull_iso_internal.h

```diff
@@ -7,3 +7,6 @@
 int ull_iso_init(void);
 int ull_iso_reset(void);
 void ull_iso_datapath_release(struct ll_iso_datapath *dp);
+void ll_iso_rx_put(memq_link_t *link, void *rx);
+void *ll_iso_rx_get(void);
+void ll_iso_rx_dequeue(void);
```

Comment:
This should be made available for LLL too when not using proprietary path.

Reply:
Similarly, `ll_rx_put` is in `ull_internal.h`. Both should probably be moved at some point. Keep for now?

Reply:
ok.